wgpu/examples/framework.rs

```diff
@@ -198,7 +198,11 @@ fn start<E: Example>(
     let spawner = Spawner::new();
     let mut config = wgpu::SurfaceConfiguration {
         usage: wgpu::TextureUsages::RENDER_ATTACHMENT,
-        format: surface.get_preferred_format(&adapter).unwrap(),
+        format: surface
```

Comment:
pop() to own the last element. Is there any implications of the change from previous behavior where the first compatible texture format was chosen?

Reply:
We should be able to do .first().unwrap()

Reply:
It returns a reference and not the owned value.

Reply:
Wait the compiler was angry at me before about it

Reply:
Just clone the enum?

![image](https://user-images.githubusercontent.com/18359644/174421077-bd92d6cd-5871-4b70-af15-bb6d74d46621.png)

Reply:
TextureFormat should be Copy (if it isn't, make it so) so you can just dereference it.